From a4caef83dc9dc52e167dd2e485c523354b1155cc Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Sun, 20 May 2012 22:26:39 +0300
Subject: [PATCH 188/370] conf_fom.c: fix formatting of dividing lines

Remove <hr> lines as they are inappropriate.

It appears that doxygen merges two /** ... */ comment blocks preceding
a function into one description. How eccentric.
---
 conf/conf_fom.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/conf/conf_fom.c b/conf/conf_fom.c
index c20fece..c5642cb 100644
--- a/conf/conf_fom.c
+++ b/conf/conf_fom.c
@@ -49,8 +49,8 @@ size_t fom_locality(const struct c2_fom *fom)
 }
 
 
-/** <hr> <!----------------------------------------------------------------------!>
- *  c2_conf_stat_resp FOM
+/* -------------------------------------------------------------------
+ * c2_conf_stat_resp FOM
  */
 
 /**
@@ -77,8 +77,8 @@ struct c2_fom_type c2_fom_stat_mopt = {
         .ft_ops = &c2_fom_stat_type_ops,
 };
 
-/** <hr> <!----------------------------------------------------------------------!>
- *  c2_conf_fetch_resp FOM
+/* -------------------------------------------------------------------
+ * c2_conf_fetch_resp FOM
  */
 
 /**
@@ -105,8 +105,8 @@ struct c2_fom_type c2_fom_fetch_mopt = {
         .ft_ops = &c2_fom_fetch_type_ops,
 };
 
-/** <hr> <!----------------------------------------------------------------------!>
- *  c2_conf_update FOM
+/* -------------------------------------------------------------------
+ * c2_conf_update FOM
  */
 
 /**
-- 
1.8.3.2

